lib/rules/no-duplicate-at-import-rules/index.mjs

```diff
@@ -9,7 +9,7 @@ import validateOptions from '../../utils/validateOptions.mjs';
 const ruleName = 'no-duplicate-at-import-rules';
 
 const messages = ruleMessages(ruleName, {
-	rejected: (atImport) => `Unexpected duplicate @import rule ${atImport}`,
+	rejected: (url) => `Unexpected duplicate @import url "${url}"`,
```

Comment:
@jeddy3 the message doesn't match the rule name anymore; do you have a suggestion?

Reply:
How about:

```
Unexpected duplicate @import with URL "${url}"
```

Reply:
```suggestion
	rejected: (url) => `Unexpected duplicate @import with URL "${url}"`,
```